tests/integration/test_board.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from src.packages.utils import Status, Platforms
+
+pytestmark = [pytest.mark.fuelrats_api, pytest.mark.asyncio, pytest.mark.integration]
+
+
+async def test_create_rescue(board_online_fx, rescue_plain_fx):
+    """ verifies a case can be created via the online board"""
+    # return Rescue(uuid4(), "UNIT_TEST", "ki", "UNIT_TEST", board_index=42, status=Status.OPEN,
+    #               platform=Platforms.PC)
+
+    async with board_online_fx.create_rescue(
+            "UNIT_TEST", "ki", "UNIT_TEST", status=Status.OPEN, platform=Platforms.PC
+    ) as rescue:
+        ...
```

Comment:
Missing functions?

Reply:
nah, Though will change in next commit since `create_rescue` is now now a context manager (see `#mechadev` )